### PR TITLE
lzfse: conan v2 support

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
`project()` before `cmake_minimum_required()` in their CMakeLists, I'll push a patch.

**Review by uilianries — APPROVED** [review]

File: recipes/lzfse/all/conandata.yml [hunk 1]
```diff
@@ -4,5 +4,4 @@ sources:
     sha256: "cf85f373f09e9177c0b21dbfbb427efaedc02d035d2aade65eb58a3cbf9ad267"
 patches:
   "1.0":
-    - patch_file: "patches/0001-fix-cmake-install-bundle.patch"
-      base_path: "source_subfolder"
+    - patch_file: "patches/0001-fix-cmake.patch"
```
Comment (prince-chrismc) [inline recipes/lzfse/all/conandata.yml:7]:
```suggestion
    - patch_file: "patches/0001-fix-cmake.patch"
       patch_type: "portability"
```

Comment (uilianries) [reply recipes/lzfse/all/conandata.yml:7]: @prince-chrismc there is an extra space.

Comment (prince-chrismc) [reply recipes/lzfse/all/conandata.yml:7]: fixed 👍

**Review by uilianries — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**